system/api/create.go

```diff
@@ -35,13 +38,201 @@ func createContentHandler(res http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	err := req.ParseMultipartForm(1024 * 1024 * 4) // maxMemory 4MB
+	switch req.Header.Get("Content-type") {
+	case "application/json":
+		createContentHandlerJSON(res, req)
+		return
+	case "multipart/form-data":
+		createContentHandlerMultiPartForm(res, req)
+		return
+	}
+}
```

Comment:
I need to handle the default case here, I think we should return 400 Bad request, on default case